metricbeat/module/kubernetes/_meta/test/docs/README.md

```diff
@@ -1,88 +1,103 @@
-# Testing Metricbeat
+## Testing Metricbeat
 
+### Prerequisites
 
-## Create kubernetes cluster using kind
-
-Follow instructions at https://kind.sigs.k8s.io/docs/user/quick-start/#installation and install kind.
-
-Create a kind kubernetes cluster.
-```
-kind create cluster  --image 'kindest/node:v1.21.1'
-```
-
-## Deploy Kube-state-metrics
-
-Prerequisite for collecting kubernetes meaningful metrics is kube-state-metrics.
-Deploy it to your cluster manually by
+Make sure that the fields are update. If you don't know how, go to metricbeat directory and run:
 ```bash
-git clone git@github.com:kubernetes/kube-state-metrics.git
-cd kube-state-metrics/
+make update
+ ```
 
-kubectl apply -k .
-```
-
-## Create ELK stack
-
-You can spin up an ELK stack in two ways
-1. [Proposed] Using elastic cloud https://cloud.elastic.co
-2. Locally on your kind cluster (EK tuple will suffice).
+Check that the modules pass the tests. Go to the directory of each of them and run:
 ```bash
-# Deploy Elasticsearch and Kibana
-cd metricbeat/module/kubernetes/_meta/test/docs
-kubectl apply -f 01_playground/ek_stack.yaml
-
-# Expose Kibana with port forwarding. In your browser visit localhost:5601
-kubectl port-forward deployment/kibana 5601:5601
-```
-
-
-## Playground Metricbeat Pod
+go test -data
+ ```
+
+
+### Deploy metricbeat
+1. Spin up the elastic stack:
+   ```bash
+   elastic-package stack up -v -dddd
+    ```
+   > Don't have elastic package installed? Find how [here](https://github.com/elastic/elastic-package/blob/main/README.md).
+
+2. Create cluster:
+    ```bash
+    kind create cluster --config=config.yaml
+    ```
+   > Don't have kind installed? Find how [here](https://kind.sigs.k8s.io/docs/user/quick-start/#installation).
+
+3. Set context:
+   ```bash
+   kubectl cluster-info --context kind-kind
+    ```
+
+4. Connect networks:
+      ```bash
+    docker network connect elastic-package-stack_default kind-control-plane
+    ```
+
+5. Deploy manifests:
+   - Testing `state_*` metricsets? Run:
+   ```bash
+   kubectl apply -k with-ksm
+    ```
+   - **Not** testing `state_*` metricsets? Run:
+   ```bash
+   kubectl apply -k without-ksm
+    ```
+   > **Note**: Adjust hosts for elasticsearch and kibana if they are not correct.
```

Comment:
I think this comment is misleading. I don't see why those hostnames should not be correct if they followed all the steps until now.

Reply:
You are right, I will remove it